sormas-api/src/main/resources/strings.properties

```diff
@@ -1521,6 +1521,8 @@ messageCustomizableEnumValueSaved = Customizable enum value saved
 messageExternalEmailAttachmentPassword=Please use this password to open the documents sent to you via email from SORMAS: %s
 messageExternalEmailAttachmentNotAvailableInfo=Attaching documents is disabled because encryption would not be possible. To encrypt documents, the person needs to have either a national health ID specified, or a primary mobile phone number set with SMS sending set up on this system.
 messagePersonNationalHealthIdInvalid=The entered national health ID does not seem to be correct
+messageSyncUsersFromAuthProviderConfigurationError=Syncing users from authentication provider is not possible because the configuration is incorrect. Please contact an admin and inform them about this issue.
```

Comment:
shouldn't have a more specific message since to this feature have access only "admin" users?